nodejs/eks/cmd/provider/cni.ts

```diff
@@ -37,6 +37,10 @@ interface VpcCniInputs {
     eniConfigLabelDef?: string;
     pluginLogLevel?: string;
     pluginLogFile?: string;
+    enablePodEni?: boolean;
+    cniConfigureRpfilter?: boolean;
+    cniCustomNetworkCfg?: boolean;
+    cniExternalSnat?: boolean;
```

Comment:
Are these meant to be exposed so users can set them? If so, we need to add them here:

https://github.com/pulumi/pulumi-eks/blob/30fafe512b3282afb6f63c9302d94e46115755e7/nodejs/eks/cni.ts#L20

And here:

https://github.com/pulumi/pulumi-eks/blob/30fafe512b3282afb6f63c9302d94e46115755e7/provider/cmd/pulumi-gen-eks/main.go#L1395-L1397

And rebuild.

Reply:
Great catch! All added and exposed in the schema as well!